### ISSUE TITLE
Wrong example

### ISSUE THREAD

**Issue by fkerekiglobant** [goldbergyoni/nodebestpractices#1255]
In [3.3 Start a Codeblock's Curly Braces on the Same Line](https://github.com/goldbergyoni/nodebestpractices#-33-start-a-codeblocks-curly-braces-on-the-same-line), the example is wrong -- the same code is shown twice.

```
// Do
function someFunction() {
  // code block
}

// Avoid
function someFunction() {
  // code block
}
```

**Comment by josephgrossberg** [comment]
Perhaps he meant this?

```js
// Do
function someFunction() {
  // code block
}

// Avoid
function someFunction() 
{
  // code block
}
```

**Comment by rluvaton** [comment]
thanks, fixed in #1258
